### PR TITLE
Include percentages in demog distributions

### PR DESCRIPTION
**Pull request by as2388**
Adapted from https://github.com/AfricasVoices/Project-OCHA/pull/83

### CODE REVIEW

File: automated_analysis.py [hunk 1]
```diff
@@ -192,25 +191,43 @@
...
+                    participants_with_opt_ins = demographic_distributions[cc.analysis_file_key][code.code_id]
+                    row = {
+                        "Demographic": cc.analysis_file_key if i == 0 else "",
+                        "Code": code.string_value,
+                        "Participants with Opt-Ins": participants_with_opt_ins,
+                    }
+
+                    # Only compute a percentage for relevant codes.
+                    if code.code_type == CodeTypes.NORMAL:
+                        row["Percent"] = round(participants_with_opt_ins / total_relevant[cc.analysis_file_key] * 100,
```
Comment (IsaackMwenda) [inline automated_analysis.py:225]:
```suggestion
                        row["Percent"] = round(participants_with_opt_ins / total_relevant[cc.analysis_file_key] * 100, 1)
```

File: automated_analysis.py [hunk 2]
```diff
@@ -192,25 +191,43 @@
...
+                                               1)
```
Comment (IsaackMwenda) [inline automated_analysis.py:226]:
```suggestion
```

**Review by IsaackMwenda — APPROVED** [review]
The code LGTM, would you be having the test output files ?

**Comment by as2388** [comment]
> The code LGTM, would you be having the test output files ?

https://drive.google.com/open?id=11L3RcObSNF73vUmS-fvN72l3w9XuvDUJ

**Comment by IsaackMwenda** [comment]
> > The code LGTM, would you be having the test output files ?
> 
> https://drive.google.com/open?id=11L3RcObSNF73vUmS-fvN72l3w9XuvDUJ

Thanks looks good to merge.

**Pull request merged**